src/libYARP_manager/src/manager.cpp

```diff
@@ -937,7 +937,8 @@ void Manager::clearExecutables()
     for(itr=runnables.begin(); itr!=runnables.end(); itr++)
     {
         // broker will be deleted by Executable
-        delete (*itr);
```

Comment:
Deleting a `nullptr` should be perfectly fine (https://stackoverflow.com/questions/4190703/is-it-safe-to-delete-a-null-pointer). Why was this necessary?

Reply:
well.. reading the link you posted it seems that is still a good practice in case of delete operator overload. but since this is not the case i'll remove it.